appengine/accessible/levelmanager.component.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * Blockly Demos: AccessibleBlockly
+ *
+ * Copyright 2016 Google Inc.
+ * https://developers.google.com/blockly/
```

Comment:
https://github.com/google/blockly-games
And all other similar files (not counting those being removed since they are copies of Accessible Blockly).